tracing-subscriber/src/filter/targets.rs

```diff
@@ -277,6 +277,31 @@ impl Targets {
         self
     }
 
+    /// Returns the explicitly set default level for this filter, if any.
+    ///
+    /// If a default level was set explicitly using [`with_default`](Self::with_default), that level
+    /// will be returned. Otherwise `None` will be returned.
```

Comment:
this will also return a `LevelFilter` if the `Targets` was parsed from a filter string that contains a level with no target. we should mention that here (and maybe also add an example showing what this method would return when the `Targets` is parsed from a string?

Reply:
Good catch, that's actually the case I'm most interested in (being able to set the default from a `Targets` parsed from user input).